applications/CompressiblePotentialFlowApplication/python_scripts/potential_flow_solver.py

```diff
@@ -1,7 +1,7 @@
 from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
 # importing the Kratos Library
 import KratosMultiphysics
-import KratosMultiphysics.ConvectionDiffusionApplication
+#import KratosMultiphysics.ConvectionDiffusionApplication
```

Comment:
remove, do not comment

Old things can always be recovered through the history of git